### PR TITLE
fix(data): update chaubis avtar ang 155-161 Pauri 9 to match source

### PR DESCRIPTION
**Pull request by Sarigpani**
### Summary of PR
Updated to match Mehron Saroop

### CODE REVIEW

**Comment by bhajneet** [comment]
I'll have to fix the chaupai and teri kripa naal sections

File: data/Sri Dasam Granth/0155.json [hunk 1]
```diff
@@ -1855,7 +1855,7 @@
         "source_line": null,
         "pronunciation": null,
         "pronunciation_information": null,
-        "type": "Sirlekh",
+        "type": "Pankti",
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0155.json:1858]: Supposed to stay `Sirlekh`

File: data/Sri Dasam Granth/0155.json [hunk 2]
```diff
@@ -1887,13 +1887,13 @@
         "translations": {
           "English": {
             "Dr. Surinder Singh Kohli": {
-              "translation": "CHAUPAI",
+              "translation": "BY THY GRACE CHAUPAI",
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0155.json:1890]: `ਤ੍ਵ ਪ੍ਰਸਾਦਿ` isnt in the text

File: data/Sri Dasam Granth/0160.json [hunk 3]
```diff
@@ -264,7 +264,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Budha Dal Mehron": "ruly qC muCM auTy iqC qIrM ]"
+          "Budha Dal Mehron": "ruly q`C m`uCM auTy iq`C qIrM  ]"
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0160.json:267]: ```mu`CM``` & remove extra space

File: data/Sri Dasam Granth/0160.json [hunk 4]
```diff
@@ -673,7 +673,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Budha Dal Mehron": "muKM muC bMkI ]"
+          "Budha Dal Mehron": "muKM m`uC bMkI ]"
```
Comment (Sarabveer) [inline data/Sri Dasam Granth/0160.json:676]: ```mu`C```

File: data/Sri Dasam Granth/0155.json [hunk 5]
```diff
@@ -3282,7 +3282,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Budha Dal Mehron": "rhw Anµq AMq nhI pwïo ]"
+          "Budha Dal Mehron": "rhw Anµq AMq nhI pwXo ]"
```
[the remark this answers is not in the document]
Comment (Sarigpani) [reply data/Sri Dasam Granth/0155.json:3285]: That is odd, the first line is correct. Ill change it.

File: data/Sri Dasam Granth/0155.json [hunk 6]
```diff
@@ -6107,7 +6107,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Budha Dal Mehron": "qw mih lIn jgiq sB hÍYbo ]34]"
+          "Budha Dal Mehron": "qw mY lIn jgiq sB hÍuYbo ]34]"
```
[the remark this answers is not in the document]
Comment (Sarigpani) [reply data/Sri Dasam Granth/0155.json:6110]: Whats the difference, looks the same in converter? Do we prefer the 2nd spelling?

image

[the remark this answers is not in the document]
Comment (Sarabveer) [reply data/Sri Dasam Granth/0155.json:6110]: You are using the converter wrong, you are converting from Unicode -> GurbaniAkhar, when the input is GurbaniAkhar

[the remark this answers is not in the document]
Comment (Sarigpani) [reply data/Sri Dasam Granth/0155.json:6110]: Ok I see, I normally do it gurbani akhar to gurbani akhar as well. So ü is always used for pairi akhar?

[the remark this answers is not in the document]
Comment (Sarabveer) [reply data/Sri Dasam Granth/0155.json:6110]: yes, a better option would be type in unicode and convert to gurbaniakhar

**Comment by Harjot1Singh** [comment]
Please update the PR title to reflect the changes made. I'll squash this one in.

**Comment by Sarigpani** [comment]
https://github.com/ShabadOS/database/issues/1788 Pending fixes.

**Comment by Sarabveer** [comment]
> 
> 
> #1788 Pending fixes.

Would be done in separate PR

**Comment by Harjot1Singh** [comment]
So no longer WIP and ready to go?

**Comment by Sarabveer** [comment]
> 
> 
> So no longer WIP and ready to go?

Yes

**Pull request merged**